src/kernels/gpu_reference_kernel/fp8_naive_conv.cpp

```diff
@@ -63,15 +63,17 @@ struct conditional<false, X, Y>
 
 template <bool predicate, typename X, typename Y>
 using conditional_t = typename conditional<predicate, X, Y>::type;
+
 } // namespace std
+
 #else
 #include <cstdint> // int8_t, int16_t
 #include <cmath>   // float_t
 #endif
+#else // __HIPCC_RTC__
+#include <limits>
 #endif // __HIPCC_RTC__
 
-#include <limits> // std::numeric_limits
```

Comment:
Actually this should be enclosed into 
```C++
#ifndef MIOPEN_DONT_USE_HIP_RUNTIME_HEADERS
#include <limits>
#endif
```
See #627. Otherwise HIP builds via COMGR (i.e. without HIPRTC) would still encounter the same problem. I'll fix this.